### DISCUSSION TITLE
Colors look wired

### DISCUSSION THREAD

**Discussion by amirhhashemi** [wezterm/wezterm#4680] [Q&A] [answered]
### What Operating System(s) are you running on?

Linux X11

### Which Wayland compositor or X11 Window manager(s) are you using?

awesomewm

### WezTerm version

20230712-072601-f4abf8fd

### Ask your question!

I'm trying to migrate from Kitty but, for some reason, the colors look different in Wezterm.

Here is my color scheme in Kitty:
![image](https://github.com/wez/wezterm/assets/87268103/1a8396dc-c10d-45a8-bdaf-937f095c046e)
And in Wezterm:
![image](https://github.com/wez/wezterm/assets/87268103/c0bcd28c-e0e6-4abc-b396-4996114a90f1)
As an example, `#c4746e` looks red-ish in Kitty which is closer to what the browser shows:
![image](https://github.com/wez/wezterm/assets/87268103/2bdd4153-55c9-450a-9545-9461ed06da48)
But in Wezterm, it looks orange.

How can I fix it? 

Full config:
```lua
local wezterm = require("wezterm")

return {
  font = wezterm.font("JetBrainsMono NF"),
  font_size = 11.0,
  freetype_load_flags = "NO_HINTING",
  window_background_opacity = 1.0,
  enable_tab_bar = false,
  bidi_enabled = true,
  window_padding = {
    left = 0,
    right = 0,
    top = 0,
    bottom = 0,
  },
  force_reverse_video_cursor = true,
  bold_brightens_ansi_colors = "No",
  colors = {
    foreground = "#c5c9c5",
    background = "#181616",

    cursor_bg = "#C8C093",
    cursor_fg = "#C8C093",
    cursor_border = "#C8C093",

    selection_fg = "#C8C093",
    selection_bg = "#2D4F67",

    ansi = {
      "#0d0c0c",
      "#c4746e",
      "#8a9a7b",
      "#c4b28a",
      "#8ba4b0",
      "#a292a3",
      "#8ea4a2",
      "#C8C093",
    },
    brights = {
      "#a6a69c",
      "#E46876",
      "#87a987",
      "#E6C384",
      "#7FB4CA",
      "#938AA9",
      "#7AA89F",
      "#c5c9c5",
    },
  },
}
```

**Comment by amirhhashemi** [comment, marked as answer]
Tmux was causing it. Adding these lines to my `tmux.conf` fixed the issue:
```
set -g default-terminal "tmux-256color"
set -ag terminal-overrides ",xterm-256color:RGB"
```
I don't know why Kitty works even without these settings.

**Reply by pencilcheck** [reply]
the changes doesn't seems to change things for me, kitty just look more vibrant whereas wezterm look dull.

The one on the left is wezterm, the one on the right is kitty.

I'm using your suggestion in my tmux

<img width="1416" alt="Screenshot 2024-05-07 at 11 20 57 PM" src="https://github.com/wez/wezterm/assets/116532/dac763b3-248d-4a3f-a88f-a67c7cee7598">

**Reply by amirhhashemi** [reply]
I found multiple different settings to enable true colors in tmux. I choose that one because it worked for me. You might want to use a different setting. Some people suggested:
```
set -g default-terminal "tmux-256color"
set -ga terminal-overrides ",*256col*:Tc"
```
A quick search should give you more options.
Also make sure:

1. You have resourced tmux after you changed the config.
2. The problem is really tmux. Check if the colors look good outside of tmux.
3. `termguicolors` is enabled in neovim if you want to test the result in neovim. Although I think it's enabled by default in the recent versions.

If the problem persists, I will re-open the discussion for you. That's all I can do.

**Reply by pencilcheck** [reply]
Thanks! The new one seems to fixed it.

<img width="1421" alt="Screenshot 2024-05-07 at 11 52 56 PM" src="https://github.com/wez/wezterm/assets/116532/f15271fd-e53a-45ba-abb8-1ca229368963">

**Reply by mvillafuertem** [reply]
@amirhhashemi  I don't understand the difference with the other options, but the only thing that worked for me is this configuration on macos

tmux.conf
```
set-option -sa terminal-features ',xterm-256color:RGB'

```
